sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/PiiEntityCategory.java

```diff
@@ -15,50 +15,50 @@
 @Immutable
 public final class PiiEntityCategory extends ExpandableStringEnum<PiiEntityCategory> {
     /** Static value ABARoutingNumber for PiiEntityCategory. */
-    public static final PiiEntityCategory ABAROUTING_NUMBER = fromString("ABARoutingNumber");
+    public static final PiiEntityCategory ABA_ROUTING_NUMBER = fromString("ABARoutingNumber");
```

Comment:
Are these not generated values? 
If this is the correct representation, we should push back the service team to make this change on their end, so that every lang does not have to make this change.

Reply:
I think probably the autogene issue. The value ABARoutingNumber looks right

Reply:
I couldn't find an equivalent of this class in the `impl` folder. 
Do you mean in the auto generated class `ABA_ROUTING_NUMBER` has the correct formatting?

Reply:
noop.  Autogenerated class still generate the value ABAROUTING_NUMBER  but the real value in `ABARoutingNumber` looks correct. So I think autogen doesn't generate the class correclty

Reply:
cc @srnagar  @jianghaolu FYI. These are seems codegen errors

Reply:
@srnagar Noticing [this generated class](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/PiiCategory.java) with incorrect casing for enum names. 

Is this a known issue for code-gen?

Reply:
The word split for these types of names can sometimes be unexpected. This is the autorest modeler issue